clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestTreeNode.java

```diff
@@ -0,0 +1,187 @@
+package com.linkedin.venice;
+
+import static com.linkedin.venice.zk.VeniceZkPaths.ADMIN_TOPIC_METADATA;
+import static com.linkedin.venice.zk.VeniceZkPaths.EXECUTION_IDS;
+import static com.linkedin.venice.zk.VeniceZkPaths.PARENT_OFFLINE_PUSHES;
+import static com.linkedin.venice.zk.VeniceZkPaths.ROUTERS;
+import static com.linkedin.venice.zk.VeniceZkPaths.STORES;
+import static com.linkedin.venice.zk.VeniceZkPaths.STORE_CONFIGS;
+import static com.linkedin.venice.zk.VeniceZkPaths.STORE_GRAVEYARD;
+
+import com.linkedin.venice.ZkCopier.TreeNode;
+import java.util.ArrayList;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class TestTreeNode {
+  private static final String CLUSTER_1 = "cluster1";
+  private static final String CLUSTER_2 = "cluster2";
+  private static final String BASE_PATH = "/venice-parent";
+  /** {@code root} is used in {@code testTreeNodeMethods()} to build the tree and then in {@code testTreeToList()} to convert the path tree to a list */
+  private TreeNode root = new TreeNode(BASE_PATH);
```

Comment:
Let's keep all tests independent so that we can run them in isolation. We can move the code to build the required tree in a function and call that in both tests